### DISCUSSION TITLE
Consider using docker when running tests

### DISCUSSION THREAD

**Discussion by SeanFarrow** [JasperFx/marten#3119] [General]
It would be really nice if Marten used Docker to spin up PostGres when running tests.

I'm happy to work on this if it is wanted.

**Comment by mysticmind** [comment]
If you look at the repo https://github.com/JasperFx/marten/blob/master/docker-compose.yml, you can use this to run Postgres in docker and run your tests.

**Comment by jeremydmiller** [comment]
I'm not sold on TestContainers if that's what you're after. Run `docker compose up -d` one time from the root of the repository, and you're good to go. I'm moving this to a discussion.
